<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerRoomsEyeViewModule</class>
 <widget class="qSlicerWidget" name="qSlicerRoomsEyeViewModule">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Room's Eye View</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="8" column="0">
    <widget class="QPushButton" name="BeamsEyeViewButton">
     <property name="text">
      <string>Beam's eye view</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="ctkCollapsibleButton" name="TransformsCollapsibleButton">
     <property name="text">
      <string>Transforms</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gantry angle:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkSliderWidget" name="CollimatorRotationSlider">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>359.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="GantryRotationSlider">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>359.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Imaging panel:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="LateralTableTopDisplacementLabel">
        <property name="text">
         <string>Table top (lateral):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="PatientSupportRotationSlider">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>179.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ctkSliderWidget" name="LongitudinalTableTopDisplacementSlider">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="ctkSliderWidget" name="LateralTableTopDisplacementSlider">
        <property name="minimum">
         <double>-230.000000000000000</double>
        </property>
        <property name="maximum">
         <double>230.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="VerticalTableTopDisplacementLabel">
        <property name="text">
         <string>Table top (vertical):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="PatientSupportRotationLabel">
        <property name="text">
         <string>Patient support rotation:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="LongitudinalTableTopDisplacementLabel">
        <property name="text">
         <string>Table top (longitudinal):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkSliderWidget" name="VerticalTableTopDisplacementSlider">
        <property name="minimum">
         <double>-500.000000000000000</double>
        </property>
        <property name="maximum">
         <double>299.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Collimator angle:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="ctkSliderWidget" name="ImagingPanelMovementSlider">
        <property name="minimum">
         <double>-68.500000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="value">
         <double>-68.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="ctkCollapsibleButton" name="AdditionalTreatmentModelsCollapsibleButton">
     <property name="text">
      <string>Additional device models</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Visibility:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ApplicatorHolderCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Applicator holder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ElectronApplicatorCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Electron applicator</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="BrainLabSRSCheckBox">
          <property name="text">
           <string>BrainLab SRS frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="LoadBasicCollimatorMountedDeviceButton">
          <property name="text">
           <string>Load basic collimator-mounted devices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LoadCustomCollimatorMountedDeviceButton">
          <property name="text">
           <string>Load custom collimator-mounted device</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="AdditionalModelsLongitudinalTranslationLabel">
          <property name="text">
           <string>Longitudinal translation:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ctkSliderWidget" name="VerticalTranslationSliderWidget"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="AdditionalModelsVerticalTranslationLabel">
          <property name="text">
           <string>Vertical translation:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="AdditionalModelsLateralTranslationLabel">
          <property name="text">
           <string>Lateral translation:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ctkSliderWidget" name="LongitudinalTranslationSliderWidget"/>
        </item>
        <item row="2" column="1">
         <widget class="ctkSliderWidget" name="LateralTranslationSliderWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="ctkCollapsibleButton" name="CollisionDetectionCollapsibleButton">
     <property name="text">
      <string>Collision detection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="CollisionsDetected">
        <property name="text">
         <string>No collisions detected.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>12</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Patient body:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLSegmentSelectorWidget" name="SegmentSelectorWidget_PatientBody"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Parameter set:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ParameterSet">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLRoomsEyeViewNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Beam:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_Beam">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLRTBeamNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="LoadTreatmentMachineButton">
       <property name="text">
        <string>Load treatment machine</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="TreatmentMachineComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerRoomsEyeViewModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ParameterSet</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerRoomsEyeViewModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_Beam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerRoomsEyeViewModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentSelectorWidget_PatientBody</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
